### PR TITLE
Add docs for billing info updates

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Release note on how to add a new email address to receive billing-related information for an org.

### CODE REVIEW

**Review by FranchommeE — APPROVED** [review]
Left a few comments with suggestions. I could be easily convinced I am wrong on those :-) , I think these are not blockers anyway.
So giving 👍

**Review by Muhammadariesta — APPROVED** [review]

File: docker-hub/release-notes.md [hunk 1]
```diff
@@ -9,6 +9,22 @@ toc_max: 2
 Here you can learn about the latest changes, new features, bug fixes, and
 known issues for each Docker Hub release.
 
+# 20201-04-13
```
Comment (FranchommeE) [inline docker-hub/release-notes.md:12]:
```suggestion
# 2021-04-13
```
Typo - remove extra 0 in the Year

**Review by FranchommeE — CHANGES_REQUESTED** [review]
Found 1 typo.
Other than that, 👍

**Review by FranchommeE — APPROVED** [review]
LGTM

**Pull request merged**